azure-pipelines.yml

```diff
@@ -36,6 +36,18 @@ variables:
   modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
 
 steps:
+- checkout: self
+  persistCredentials: true
+  clean: true
+
+- task: ShellScript@2
+  displayName: Validate GitOps pipeline
+  inputs:
+    scriptPath: gitops/azure-devops/build.sh
```

Comment:
Which branch is this from?

Reply:
Not sure if I understand, could you elaborate?